stock_packaging_calculator/models/product.py

```diff
@@ -145,3 +149,57 @@ def _prepare_qty_by_packaging_values(self, packaging, qty_per_pkg):
             "is_unit": packaging.is_unit,
             "barcode": packaging.barcode,
         }
+
+    def product_qty_by_packaging_as_str(self, prod_qty, include_total_units=False):
+        """Return a string representing the qty of each packaging.
+
+        :param prod_qty: the qty of current product to translate to pkg qty
+        :param include_total_units: includes total qty required initially
+        """
+        self.ensure_one()
+        if not prod_qty:
+            return ""
+
+        qty_by_packaging = self.product_qty_by_packaging(prod_qty)
+        if not qty_by_packaging:
+            return ""
+
+        # Exclude unit qty and reuse it later
+        unit_qty = None
+        has_only_units = True
+        _qty_by_packaging = []
+        for pkg_qty in qty_by_packaging:
+            if pkg_qty["is_unit"]:
+                unit_qty = pkg_qty["qty"]
+                continue
```

Comment:
what if you have more than 1 packaging with units? Is it even possible?
Shouldn't we set "unit_qty" as 0 and sum the qties instead?

Reply:
impossible. The unit is added explicitly at the end of the pkg list and it's always the product uom.

Reply:
Okay then